orchestrationSpecs/packages/migration-workflow-templates/src/resourceLoader.ts

```diff
@@ -1,8 +1,13 @@
 import * as fs from "node:fs";
 import path from "node:path";
 
-const scriptDir = path.join(__dirname,'..','resources', 'configManagementHelpers');
+const configManagementHelpersDir = path.join(__dirname,'..','resources', 'configManagementHelpers');
 
-export const initTlhScript = fs.readFileSync(path.join(scriptDir, 'init.sh'), 'utf8');
-export const decrementTlhScript = fs.readFileSync(path.join(scriptDir, 'decrement.sh'), 'utf8');
-export const cleanupTlhScript = fs.readFileSync(path.join(scriptDir, 'cleanup.sh'), 'utf8');
+export const initTlhScript = fs.readFileSync(path.join(configManagementHelpersDir, 'init.sh'), 'utf8');
+export const decrementTlhScript = fs.readFileSync(path.join(configManagementHelpersDir, 'decrement.sh'), 'utf8');
+export const cleanupTlhScript = fs.readFileSync(path.join(configManagementHelpersDir, 'cleanup.sh'), 'utf8');
+
+const testMigrationHelpersDir = path.join(__dirname,'..','resources', 'testMigrationHelpers');
+
+export const configureAndSubmitScript = fs.readFileSync(path.join(testMigrationHelpersDir, 'configureAndSubmit.sh'), 'utf8');
+export const monitorScript = fs.readFileSync(path.join(testMigrationHelpersDir, 'monitor.sh'), 'utf8');
```

Comment:
It might be cleaner for eventual separation to have a resourceLoader for configurationManagement (or whatever you want to call that) and test.